programs/bpf/tests/programs.rs

```diff
@@ -571,10 +571,10 @@ fn assert_instruction_count() {
             ("solana_bpf_rust_128bit", 543),
             ("solana_bpf_rust_alloc", 19082),
             ("solana_bpf_rust_dep_crate", 2),
-            ("solana_bpf_rust_external_spend", 465),
+            ("solana_bpf_rust_external_spend", 473),
```

Comment:
Only thing I can see here that might have caused this are the changes in `programs/bpf/Cargo.lock`